### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#430] [Esami & Esoneri M-Z]
## Traccia

<img width="978" alt="image" src="https://github.com/user-attachments/assets/da9b7074-37d2-4b33-a0f3-e6d9421ebf25">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/603a6ba9-2d95-4b96-9252-92cfad9ea04a)
